vcr/pe/types.go

```diff
@@ -69,13 +72,16 @@ type IsHolderItems struct {
 
 // PresentationDefinition
 type PresentationDefinition struct {
-	Format                 *PresentationDefinitionClaimFormatDesignations `json:"format,omitempty"`
-	Frame                  *Frame                                         `json:"frame,omitempty"`
-	Id                     string                                         `json:"id"`
-	InputDescriptors       []*InputDescriptor                             `json:"input_descriptors"`
-	Name                   string                                         `json:"name,omitempty"`
-	Purpose                string                                         `json:"purpose,omitempty"`
-	SubmissionRequirements []*SubmissionRequirement                       `json:"submission_requirements,omitempty"`
+	Format *PresentationDefinitionClaimFormatDesignations `json:"format,omitempty"`
+	Frame  *Frame                                         `json:"frame,omitempty"`
+	// Id is the id of the presentation definition, it must be unique within the context.
+	Id               string             `json:"id"`
```

Comment:
Not sure how it's used. You have to reference it when creating a presentation submission. In that case it would not matter if it has changed at some point.